presubmit.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+set -x
+
+find . -name "*.pb.go" -exec rm {} \;
+find . -name "mock_*" -exec rm {} \;
+golint --set_exit_status ./...
+go vet ./...
+go generate -run="protoc" ./...
+go generate -run="mockgen" ./...
+go build ./...
+go test -cover ./...
+# errcheck ./...
+find . -type d -exec gocyclo -over 25 {} \;
```

Comment:
I don't have either of those binaries (gocyclo, misspell). Could you please add a section presubmits at the top README.md, including how to get their dependencies.